### ISSUE TITLE
AWS Icon not working

### ISSUE THREAD

**Issue by salehkaddoura** [vorillaz/devicons#92]
Using the CDN 1.8.0  and AWS icon not working. Seems to be missing from the .css file

**Comment by sergiorb** [comment]
yeah, the problem persist today... you have to download and install manually.

**Comment by vorillaz** [comment]
🫡 https://devicons.io/icons/aws/
